### ISSUE TITLE
Some categorical ticks are plotted as float numbers with `optuna.visualization.plot_contour`

### ISSUE THREAD

**Issue by g-votte** [optuna/optuna#2242]
With `optuna.visualization.plot_contour`, some categorical ticks are shown as float numbers. 

## Environment

- Optuna version: 2.4.0
- Python version: 3.7.9

## Reproducible examples

Run the following code:
```python
import optuna

def objective(trial):
    trial.suggest_categorical("a", ["one", "two", "three"])
    trial.suggest_categorical("b", ["eins", "zwei", "drei"])
    trial.suggest_categorical("c", ["uno", "dos", "tres"])
    return 0

study = optuna.create_study()
study.optimize(objective, n_trials=50)

optuna.visualization.plot_contour(study)
```

## Current behavior
The following figure shows the output of the repro example. Some ticks are not shown as categorical values. For example the values of `a` in the vertical axis and those of `a` `c` in the horizontal axis are float numbers.

![image](https://user-images.githubusercontent.com/1061922/105135223-02996880-5b33-11eb-8658-cd54e6fde1b9.png)

## Expected behavior
All categorical parameters are shown with their categorical labels.

**Comment by nzw0301** [comment]
hi, @g-votte , I think this issue is solved by #2297. Could you close the issue? If there is a thing for this issue, I'm happy to work for it.

**Comment by g-votte** [comment]
@nzw0301 
Closing the issue. Thanks for your contribution!
